README.md

```diff
@@ -27,6 +27,14 @@ creating notes and occurrences in Grafeas. There are client libraries available
 * The authoritative API for grafeas is the [protobuf
 files](https://github.com/Grafeas/Grafeas/tree/master/proto/v1beta1).
 
+## Storage
```

Comment:
nit: let's rename this section to Storage Backends, for clarity

Reply:
Done!